fat-jar.html

```diff
@@ -0,0 +1,321 @@
+<p>
+    Finally, you can also create a runnable fat jar for your JavaFX 11 project on
+    command line.
+</p>
+
+<div class="alert alert-warning">
+    <strong>Warning</strong>: This is a discouraged tedious error-prone manual process 
+    that should be avoided by using the <a href="#Maven">Maven</a>'s shade plugin or 
+    the <a href="#Gradle">Gradle</a>'s jar task, in case <a href="index.html#modular">jlink</a>
+    is not applicable.
+</div>
+
+<p>
+    An example can be found in this  
+    <a href="https://github.com/openjfx/samples/tree/master/CommandLine/Non-modular/CLI" target="_blank">project</a>.
+    
+    Download the appropriate
+    <a target="_blank" href="https://gluonhq.com/products/javafx/">JavaFX SDK</a>
+    for your operating system and unzip it to a desired location.
+</p>
+
+<p>
+    Add an environment variable pointing to the lib directory of the runtime:
+</p>
+
+<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <a class="nav-link active" href="#nix-fxpath" data-toggle="tab">Linux/Mac</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#win-fxpath" data-toggle="tab">Windows</a>
+    </li>
+</ul>
+
+<div class="tab-content">
+    <div class="tab-pane active" id="nix-fxpath">
+<pre class="no-border-radius"><code>
+export PATH_TO_FX=/path/to/javafx-sdk-11/lib
+</code></pre>
+    </div>
+    <div class="tab-pane" id="win-fxpath">
+<pre><code>
+set PATH_TO_FX="path\to\javafx-sdk-11\lib"
+</code></pre>
+    </div>
+</div>
+
+<p>
+    Compile the project:
+</p>
+
+<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <a class="nav-link active" href="#nix-javacout" data-toggle="tab">Linux/Mac</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#win-javacout" data-toggle="tab">Windows</a>
+    </li>
+</ul>
+
+<div class="tab-content">
+    <div class="tab-pane active" id="nix-javacout">
+<pre class="no-border-radius"><code>
+javac --module-path $PATH_TO_FX --add-modules=javafx.controls -d out $(find src/main/java -name "*.java")
+</code></pre>
+    </div>
+    <div class="tab-pane" id="win-javacout">
+<pre><code>
+dir /s /b src\main\java\*.java > sources.txt & javac --module-path %PATH_TO_FX% --add-modules=javafx.controls -d out @sources.txt & del sources.txt
+</code></pre>
+    </div>
+</div>
+
+<p>
+    And create the fat jar, adding the JavaFX required jars and the platform-specific native libraries. For 
+    the <kbd>hellofx</kbd> project:
+</p>
+
+<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <a class="nav-link active" href="#nix-jar" data-toggle="tab">Linux/Mac</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#win-jar" data-toggle="tab">Windows</a>
+    </li>
+</ul>
+
+<div class="tab-content">
+    <div class="tab-pane active" id="nix-jar">
+<pre class="no-border-radius"><code>
+find $PATH_TO_FX/{javafx.base.jar,javafx.graphics.jar,javafx.controls.jar} -exec unzip -nq {} -d out \;
+
+#uncomment for Linux:
```

Comment:
If the linux and mac commands are going to be different, I would propose to have a separate tab for linux and mac. This probably requires all other command line blocks to be split as well though.

Reply:
That's a big issue, there are too many tab blocks, and this will imply duplicating them. But if that's required, it can be done of course.